packages/card/index.js

```diff
@@ -78,8 +88,7 @@ class WarpCard extends WarpElement {
   render() {
     return html`
       <div tabindex=${ifDefined(this.clickable ? '0' : undefined)} class="${this._containerClasses}" @keydown=${this.keypressed}>
-        ${this.clickable ? this.uuButton : ''} ${!this.clickable && this.selected ? this.uuSpan : ''}
-        <div class="${this._outlineClasses}"></div>
+        ${this._interactiveElement} ${!this.flat ? html`<div class="${this._outlineClasses}"></div>` : ''}
```

Comment:
Wasn't `<div class="${this._outlineClasses}"></div>` added irrespective of if this.flat was true or not? 🤔 

Also this is cleaner I think:
`${this.flat ? '' : html`<div class="${this._outlineClasses}"></div>`}`

Reply:
In react and vue we only render this div with the outline-classes if the flat prop is false. So I assumed we wanted the same behavior for elements? Otherwise if it should be rendered regardless of whether the flat prop is true or not, then we need to change that in react and vue as well.

Reply:
It will look like this if we don't check if this.flat is false: 
<img width="1119" alt="Skärmavbild 2024-07-31 kl  14 48 58" src="https://github.com/user-attachments/assets/5c335b58-a5ed-42dd-a821-06c0eb0cb055">

and it will look like this if we do check: 
<img width="1109" alt="Skärmavbild 2024-07-31 kl  14 49 07" src="https://github.com/user-attachments/assets/68dfbb37-d97d-48c7-89dc-31d3c2c40ce4">

Reply:
I do feel however that the outline is more correct in the first example. I mean then the outline is behaving like an outline should behave! :D But we just need to decide on what we want and make sure it looks and behaves the same in react, vue and elements :)